src/js/base/ripe.js

```diff
@@ -153,22 +157,33 @@ ripe.Ripe.prototype.setOptions = function(options) {
 };
 
 ripe.Ripe.prototype.setPart = function(part, material, color, noUpdate) {
-    this._setPart(part, material, color, true);
+    if (noUpdate !== true) {

```

Comment:
why this condition ?

Reply:
the `noUpdate` is now a bit outdated in the sense that if one wants to call the no update it's better off calling just the `_set_part` right ?

maybe it's better to just do

```
if(noUpdate) {
    return this._setPart(part, material, color);
}
```

at the beginning of the function